pkg/scheduler/metrics/metrics.go

```diff
@@ -23,70 +23,105 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const schedulerSubsystem = "scheduler"
+const (
+	// SchedulerSubsystem - subsystem name used by scheduler
+	SchedulerSubsystem = "scheduler"
+	// SchedulingLatencyName - scheduler latency metric name
+	SchedulingLatencyName = "scheduling_latency_seconds"
+
+	// OperationLabel - operation label name
+	OperationLabel = "operation"
+	// Binding - binding operation label value
+	Binding = "binding"
+	// SchedulingAlgorithm - scheduling algorithm operation label value
+	SchedulingAlgorithm = "selecting_node"
```

Comment:
Please rename the variable to match the string.

Reply:
done